ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -614,7 +617,10 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
                 log.error("Error saving the download: \(error)")
                 throw KMPError.copyFiles
               }
-              Manager.shared.addKeyboard(keyboard)
+              if !haveInstalledOne {
```

Comment:
Why are you only installing a single keyboard from the package? We want to install all the keyboards in the package, but only install a single language for each keyboard.

Reply:
As far as I can tell, `haveInstalledOne` is still going to block multiple keyboards from installing?

Reply:
I don't think so. the line "var haveInstalledOne = false" is *inside* the "k in keyboards" loop, which loops over all the keyboards. What it is preventing is multiple (language, keyboard) pairs from being added to preferences. Just the first one gets added, but all the files get installed.

Reply:
My bad ;-)